gtsam/CMakeLists.txt

```diff
@@ -89,7 +89,11 @@ list(APPEND gtsam_srcs "${PROJECT_BINARY_DIR}/config.h" "${PROJECT_BINARY_DIR}/d
 install(FILES "${PROJECT_BINARY_DIR}/config.h" "${PROJECT_BINARY_DIR}/dllexport.h" DESTINATION include/gtsam)
 
 if(GTSAM_SUPPORT_NESTED_DISSECTION)
-    list(APPEND GTSAM_ADDITIONAL_LIBRARIES metis)
+    if(GTSAM_USE_SYSTEM_METIS)
+        list(APPEND GTSAM_ADDITIONAL_LIBRARIES ${METIS_LIBRARIES})
```

Comment:
Hmmm. Can we not make it so that METIS_LIBRARIES points to internal *if* we use the internal version?

Reply:
Yeah I think I can do that.